src/commands/General/mergepdf.ts

```diff
@@ -0,0 +1,45 @@
+import { ApplyOptions } from '@sapphire/decorators';
+import type { Args, CommandOptions, UserError } from '@sapphire/framework';
+import PdfMerger from 'pdf-merger-js';
+import pupa from 'pupa';
+import { mergePDF as config } from '@/config/commands/general';
+import MonkaCommand from '@/structures/MonkaCommand';
+import type { GuildMessage } from '@/types';
+
+@ApplyOptions<CommandOptions>(config.options)
+export default class PdfMergeCommand extends MonkaCommand {
+  public async run(message: GuildMessage, args: Args): Promise<void> {
+    // Argument dans l'ordre croissant
+
+    const msgIds: string[] = [];
+    while (!args.finished) {
+        const msgId = await args.pickResult('string');
+        if (msgId.success) {
+            msgIds.push(msgId.value);
+        } else {
+            const { parameter } = (config.messages.noPdfFound as UserError & { parameter: string });
+            await message.channel.send(pupa('{pdf_Error}', { pdfError: parameter }));
+            return;
+        }
+    }
+
+    const merger = new PdfMerger();
+
+    for (const o of msgIds) {
+        const msg = message.channel.messages.cache.get(o);
+        for (const a of msg.attachments.values()) {
+            if (a.name.includes('.pdf'))
```

Comment:
```suggestion
            if (a.name.endsWith('.pdf'))
```
Encore plus pratique :)